jobs/github-event/marketplace_purchase/cancelled.js

```diff
@@ -0,0 +1,11 @@
+const dbs = require('../../../lib/dbs')
+const upsert = require('../../../lib/upsert')
+
+module.exports = async function ({ marketplace_purchase }) {
+  const { payments } = await dbs()
+  const accountId = String(marketplace_purchase.account.id)
+
+  await upsert(payments, accountId, {
+    plan: 'free'
```

Comment:
Is this when someone moves from a paid GH plan to free, or when an installation is removed? If it’s the former, shouldn’t we be moving them to the `opensource` plan instead of `free`?

Reply:
This happens when they cancel their Marketplace purchase. And since the 'opensource' plan is also technically a Marketplace (but free of change) purchase I thought it would fall back to the regular 'free' plan

Reply:
Yeah, that’s exactly what I was wondering… not sure on this either, need to make a product decision @janl 

I think I’d prefer leaving people inside the ecosystem they’re in, just for consistency. It doesn’t make a difference functionally, at least I don’t think so.

Reply:
If I read everything correctly, then:
- If they keep the installation but go from `team` or `business` to unpaid, it should be `free`
- if they remove the installation, they should be nothing.